src/status_im2/contexts/profile/profiles/views_v2.cljs

```diff
@@ -0,0 +1,123 @@
+(ns status-im2.contexts.profile.profiles.views-v2
+  (:require [clojure.string :as string]
+            [quo.core :as quo]
+            [quo.design-system.colors :as colors]
+            [re-frame.core :as re-frame]
+            [reagent.core :as reagent]
+            [status-im.ui.components.copyable-text :as copyable-text]
+            [status-im.ui.components.list-selection :as list-selection]
+            [status-im.ui.components.react :as react]
+            [status-im2.contexts.profile.profiles.components :as components]
+            [status-im2.contexts.profile.profiles.list-items :as profile-list-item]
+            [status-im2.contexts.profile.profiles.style :as styles]
+            [status-im.utils.universal-links.utils :as universal-links]
+            [status-im2.common.qr-code-viewer.view :as qr-code-viewer]
+            [status-im2.config :as config]
+            [utils.i18n :as i18n])
+  (:require-macros [status-im.utils.views :as views]))
+
+
+(views/defview share-chat-key
+  []
+  (views/letsubs [{:keys [address ens-name]} [:popover/popover]
```

Comment:
hmm @flexsurfer is there better sub for @nguyentruongky to get this address & ens-name? 🤔